vercel/resource_project_environment_variable.go

```diff
@@ -0,0 +1,239 @@
+package vercel
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/provider"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/vercel/terraform-provider-vercel/client"
+)
+
+type resourceProjectEnvironmentVariableType struct{}
+
+// GetSchema returns the schema information for an project environment variable resource.
+func (r resourceProjectEnvironmentVariableType) GetSchema(_ context.Context) (tfsdk.Schema, diag.Diagnostics) {
+	return tfsdk.Schema{
+		Description: `
+Provides a Project Environment Variable resource.
+
+An Project Environment Variable resource allows to add an environment variable to a Vercel project `,
```

Comment:
`allows to add` doesn't make sense gramatically here. Could we also link to the Vercel documentation for Environment Variables? https://vercel.com/docs/concepts/projects/environment-variables